### PR TITLE
Fix compile time error when compiled using Kernel 6.1.0+ versions

### PR DESCRIPTION
**Pull request by rajkumar-xilinx**
Compile time errors seen on Kernel 6.1.0+ versions.
On latest kernels from 6.1.0+ versions, set_termios function is declared as below
void (set_termios*)(struct uart_port *, struct ktermios *, const struct ktermios *)

On older kernels upto 6.0 versions, set_termios function is declared as below
void (set_termios*)(struct uart_port *, struct ktermios *, struct ktermios *)

This PR fixes this issue.

#### Problem solved by the commit
Fixed compile time warnings and errors seen when XRT compiled using Kernel 6.0.0+
#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
Internal finding
#### How problem was solved, alternative solutions (if any) and why they were rejected
Fixed compile time warnings and errors seen when XRT compiled using Kernel 6.0.0+
#### Risks (if any) associated the changes in the commit
NA
#### What has been tested and how, request additional testing if necessary
XRT compilation on kernel 6.0.0+ versions.
#### Documentation impact (if any)
NA

### CODE REVIEW

**Review by chvamshi-xilinx — APPROVED** [review]

**Review by maxzhen — APPROVED** [review]

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/ulite.c [hunk 1]
```diff
@@ -339,7 +339,7 @@ static void ulite_shutdown(struct uart_port *port)
 }
 
 static void ulite_set_termios(struct uart_port *port, struct ktermios *termios,
-			      struct ktermios *old)
+			      const struct ktermios *old)
```
Comment (keryell) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/ulite.c:342]: Please avoid mixing tabs and spaces.
https://addons.mozilla.org/en-US/firefox/addon/refined-github-/ is showing this to me.

**Comment by chvamshi-xilinx** [comment]
retest this please

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/ulite.c (hunk 1 above)
Comment (rajkumar-xilinx) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/ulite.c:346]: Hi @keryell Thanks for the review. I have fixed it, please review and let me know.  Thank you.

**Comment by rajkumar-xilinx** [comment]
retest this please

**Review by keryell — APPROVED** [review]
Thanks.

**Comment by dayeh-xilinx** [comment]
retest this please.

**Comment by dayeh-xilinx** [comment]
retest this please.

**Pull request merged**